src/libs/resizing-image-on-demand/FastImage.tsx

```diff
@@ -8,6 +8,7 @@ type Props = Omit<FastImageProps, 'source'> & {
   url: string
   height?: number
   width?: number
+  alt?: string
```

Comment:
#en cas d'erreur avec l'image, il me semble que l'affichage était mieux lorsqu'on avait `alt=""` que losqu'on avait pas de `alt`

je suggère de rendre l'obligatoire cette propriété et forcer le dev qui utilise une `FastImage` à se poser la question "est-ce que cette image apporte une information ?"

```suggestion
  alt: string
```

ou alors carrément forcer via le typage à se poser la question

```ts
type IllustrativeImage = {
  alt?: never
  illustrativeImage: true
}

type InformativeImage = {
  alt: string
}

type IllustrativeOrInformativeImage = IllustrativeImage || InformativeImage

type Props = Omit<FastImageProps, 'source'> & {
  url: string
  height?: number
  width?: number
  children?: never
} & IllustrativeOrInformativeImage
```